clients/parity:master/parity.sh

```diff
@@ -109,8 +109,9 @@ if [ "$HIVE_FORK_BYZANTIUM" != "" ]; then
 #	chainconfig=`echo $chainconfig | jq "setpath([\"params\", \"eip98Transition\"]; \"0x$HIVE_FORK_BYZANTIUM\")"`
 
 	# Ethash params
-	chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"eip649Reward\"]; \"0x29A2241AF62C0000\")"`
-	chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"eip649Transition\"]; \"0x$HIVE_FORK_BYZANTIUM\")"`
+	#  Set blockreward to 3M for byzantium
+	chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"blockReward\",\"0x$HIVE_FORK_BYZANTIUM\"]; \"0x29A2241AF62C0000\")"`
+	# difficulty calculation -- aka bomb delay
```

Comment:
For the bomb delay we also need to define additional variables : 
```
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"difficultyBombDelays\",\"0x$HIVE_FORK_BYZANTIUM\"]; \"3000000\")"`
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"difficultyBombDelays\",\"0x$HIVE_FORK_CONSTANTINOPLE\"]; \"5000000\")"`

```

Reply:
I'm not sure that'll make a difference -- on hive there are only a couple of blocks, so whether the bomb is delayed to 3M or 5M shouldn't matter. Hive can't really be used to test the bomb delay, I think

Reply:
Also added an empty difficultyBombDelays struct to chain.json

Reply:
Hello, I made a small (big) mistake, our 'difficultyBombDelays' is additive so the right code should be 
```
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"difficultyBombDelays\",\"0x$HIVE_FORK_BYZANTIUM\"]; \"3000000\")"`
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", "difficultyBombDelays\",\"0x$HIVE_FORK_CONSTANTINOPLE\"]; \"2000000\")"`
```
to have 5M total delay at constantinople.